tools/cosmovisor/cmd/cosmovisor/add_upgrade.go

```diff
@@ -110,7 +110,7 @@ func saveOrAbort(path string, data []byte, force bool) error {
 		return fmt.Errorf("failed to check if file exists: %w", err)
 	}
 
-	if err := os.WriteFile(path, data, 0o600); err != nil {
+	if err := os.WriteFile(path, data, 0o755); err != nil {
```

Comment:
Can you add a changelog in cosmovisor and a nolint comment so we don't revert it back because of the gosec? And so that linting passes?

Reply:
```
cmd/cosmovisor/add_upgrade.go:58:2: directive `//nolint:gosec` is unused for linter "gosec" (nolintlint)
	//nolint:gosec
	^
cmd/cosmovisor/add_upgrade.go:115:2: directive `//nolint:gosec` should provide explanation such as `//nolint:gosec // this is why` (nolintlint)
	//nolint:gosec
	^
```

lol now it gives me this. do you think it's worth disabling? or what's better to do here?

Reply:
nevermind, I fixed it

Reply:
and also added the changelog entry. @julienrbrt can you check?